src/acme/ui/CustomerListUI.java

```diff
@@ -0,0 +1,319 @@
+    lblCustomers.setFont(new Font("Lucida Grande", Font.PLAIN, 24));
+    lblCustomers.setBounds(20, 20, 200, 27);
+    add(lblCustomers);
+    
+    //my test data
+    Customer[] testData = getTestData();
+    
+
+    //scroll and table. I made this in netbeans, so it looks a little different
+    JScrollPane jScrollPane1 = new JScrollPane();
+    JTable jTable1 = new JTable();
+
+    //set up table
+    jTable1.setBorder(javax.swing.BorderFactory.createEtchedBorder());
+    jTable1.setFont(new java.awt.Font("Lucida Grande", 0, 16)); 
+    jTable1.setModel(new javax.swing.table.DefaultTableModel(
+        new Object [][] {
+            {testData[0].getCustomerNumber(), testData[0].getName(), testData[0].getIntersection().getIntersectionName(), testData[0].isActive(),'\u270E' + " "+ testData[0].getName()},
+            {testData[1].getCustomerNumber(), testData[1].getName(), testData[1].getIntersection().getIntersectionName(), testData[1].isActive(),'\u270E' + " "+ testData[1].getName()},
+            {testData[2].getCustomerNumber(), testData[2].getName(), testData[2].getIntersection().getIntersectionName(), testData[2].isActive(),'\u270E' + " "+ testData[2].getName()},
+            {testData[3].getCustomerNumber(), testData[3].getName(), testData[3].getIntersection().getIntersectionName(), testData[3].isActive(),'\u270E' + " "+ testData[3].getName()},
+            {testData[4].getCustomerNumber(), testData[4].getName(), testData[4].getIntersection().getIntersectionName(), testData[4].isActive(),'\u270E' + " "+ testData[4].getName()}
+        },
+        new String [] {
+            "Number", "Name", "Location", "Active", "Edit"
+        }
+    ) {
+        Class[] types = new Class [] {
+            java.lang.String.class, java.lang.String.class, java.lang.String.class, java.lang.Boolean.class, javax.swing.JButton.class
+        };
+        boolean[] canEdit = new boolean [] {
+            //need last one true so can click button
+            false, false, false, false, true
+        };
+
+        public Class getColumnClass(int columnIndex) {
+            return types [columnIndex];
+        }
+
+        public boolean isCellEditable(int rowIndex, int columnIndex) {
+            return canEdit [columnIndex];
+        }
+    });
+    
+    //needed for buttons in table
+    TableColumn editCol = jTable1.getColumnModel().getColumn(4);
+    JButton editButton = new JButton("edit");
+    editCol.setCellRenderer(new ButtonRenderer());;
+    editCol.setCellEditor(new ButtonEditor(new JTextField()));
+  
+    //more table properties
+    jTable1.setColumnSelectionAllowed(false);
+    jTable1.getTableHeader().setFont(new Font("Lucida Grande", 0, 18)); 
+    jTable1.setRowHeight(24);
+    jTable1.setRowMargin(2);
+    jTable1.setSelectionBackground(new Color(93, 184, 202));
+    jTable1.setSelectionForeground(new java.awt.Color(0, 0, 0));
+    jTable1.setShowGrid(true);
+    jScrollPane1.setViewportView(jTable1);
```

Comment:
Can you make this pane taller?  It's kinda short

Reply:
I made it taller.